src/dotnet/install.sh

```diff
@@ -4,47 +4,22 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 #
-# Docs: https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/dotnet.md
-# Maintainer: The VS Code and Codespaces Teams
-
+# Docs: https://github.com/devcontainers/features/tree/main/src/dotnet
+# Maintainer: The Dev Container spec maintainers
 DOTNET_VERSION="${VERSION:-"latest"}"
-DOTNET_RUNTIME_ONLY="${RUNTIMEONLY:-"false"}"
-OVERRIDE_DEFAULT_VERSION="${OVERRIDEDEFAULTVERSION:-"true"}"
-INSTALL_USING_APT="${INSTALLUSINGAPT:-"true"}"
-
-DOTNET_LATEST="7"
-DOTNET_LTS="6"
-
-USERNAME="${USERNAME:-"${_REMOTE_USER:-"automatic"}"}"
-UPDATE_RC="${UPDATE_RC:-"true"}"
-TARGET_DOTNET_ROOT="${TARGET_DOTNET_ROOT:-"/usr/local/dotnet"}"
-ACCESS_GROUP="${ACCESS_GROUP:-"dotnet"}"
-
-MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
-DOTNET_ARCHIVE_ARCHITECTURES="amd64"
-DOTNET_ARCHIVE_VERSION_CODENAMES="buster bullseye bionic focal hirsute jammy"
-# Feed URI sourced from the official dotnet-install.sh
-# https://github.com/dotnet/install-scripts/blob/1b98b94a6f6d81cc4845eb88e0195fac67caa0a6/src/dotnet-install.sh#L1342-L1343
-DOTNET_CDN_FEED_URI="https://dotnetcli.azureedge.net"
-# Ubuntu 22.04 and on do not ship with libssl1.1, which is required for versions of .NET < 6.0
-DOTNET_VERSION_CODENAMES_REQUIRE_OLDER_LIBSSL_1="buster bullseye bionic focal hirsute"
+ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS:-""}"
```

Comment:
```suggestion
ADDITIONAL_VERSIONS="${ADDITIONALVERSIONS}"
```

I understand that you are trying to follow the old semantics, however, updating all such options would help sync the `ADDITIONAL_VERSIONS` var with the Feature defaults.

eg. 

```
"additionalVersions": {
            "type": "string",
            "default": "",
            "description": "Enter additional .NET SDK versions, separated by commas. Use 'latest' for the latest version, 'lts' for the latest LTS version, 'X.Y' or 'X.Y.Z' for a specific version."
        },
```

Hence, the CLI always sets `ADDITIONALVERSIONS` to "".  See https://containers.dev/implementors/features/#options-property Hence, this is no longer needed and its adds unwanted maintainence. eg. Updating default value of `additionalVersions` would always need a change to this line.

Reply:
@samruddhikhandale does this comment also apply to `DOTNET_VERSION="${VERSION:-"latest"}"`? That's where added it back to make sure an empty string behaves the same as the default.

Reply:
> The CLI does not use the default for version in this case.

I think that's the correct way ; whatever the user passes should be taken as input and we should only consider defaults when the option is omitted.

Reply:
Right, just to be clear, I can keep the fallback to `latest` for the version option?

``` sh
DOTNET_VERSION="${VERSION:-"latest"}"
```

Otherwise if you specify empty string for the version option then you get a technical error.

``` sh
12.58 Executing scripts/vendor/dotnet-install.sh --version  --channel  --install-dir /usr/share/dotnet --no-path
16.67 scripts/vendor/dotnet-install.sh: line 1253: effective_versions[$link_index]: unbound variable
16.68 ERROR: Feature "Dotnet CLI" (Unknown) failed to install! Look at the documentation at https://github.com/devcontainers/features/tree/main/src/dotnet for help troubleshooting this error.
```

Reply:
Yes, I think this is fine for now 👍